scripts/aws/setupdb.sh

```diff
@@ -19,5 +19,10 @@ envdir /etc/nyc-trees.d/env /opt/app/manage.py migrate
 # Load block face data
 envdir /etc/nyc-trees.d/env /opt/app/manage.py loaddata $DIR/../src/nyc_trees/apps/survey/fixtures/blockface.json
 
+# Setup soft launch flag if it does not already exist
+if [ envdir /etc/nyc-trees.d/env /opt/app/manage.py flag -l | grep full_access ]; then
+    envdir /etc/nyc-trees.d/env /opt/app/manage.py flag full_access --create --deactivate
```

Comment:
Including ``--deactivate`` sets the ``everyone`` field to ``false`` which means that the flag will never take effect for anyone. We need to omit this option so that the flag will work for the users who should have full access.